### PR TITLE
Refactor 1e stats to allow for monsters from all books

### PR DESCRIPTION
**Pull request by verdammelt**
monster_list.csv becomes mm.csv and empty ff.csv and mm2.csv added.

### CODE REVIEW

**Review by tunesmith — APPROVED** [review]
I don't fully understand all the Makefile changes, but I checked out the PR locally and rebuilt, and my modules still load monsters, so it looks good. :)

File: rpg-1e-stats/Makefile [hunk 1]
```diff
@@ -1,27 +1,30 @@
-#TARGET=test_gnoll.pdf
 TARGET=test_all.pdf
 
 all: $(TARGET)
 
 show: $(TARGET)
 	evince $(TARGET)
 
+$(TARGET): rpg-1e-stats.def
```
Comment (verdammelt) [inline rpg-1e-stats/Makefile:8]: target file needs def file but did not depend upon it previously.

File: rpg-1e-stats/Makefile [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+
 %.pdf: %.tex
 	pdflatex $<
 
-convert:
-	./convert
+%.def: %.csv convert
```
Comment (verdammelt) [inline rpg-1e-stats/Makefile:13]: `ANYTHING.def` depends upon `ANYTHING.csv` and `convert` and then how to do it.

File: rpg-1e-stats/Makefile [hunk 3]
```diff
@@ -1,27 +1,30 @@
...
+	./convert $<
 
-rpg-1e-stats.def: convert
+rpg-1e-stats.csv: mm.csv ff.csv mm2.csv
```
Comment (verdammelt) [inline rpg-1e-stats/Makefile:16]: rpg-1e-stats.csv depends upon these three files and how to put them together to do it!

File: rpg-1e-stats/Makefile [hunk 4]
```diff
@@ -1,27 +1,30 @@
...
+	tail -n +3 ff.csv >> $@
+	tail -n +3 mm2.csv >> $@
 
 install: rpg-1e-stats.def
 	cp rpg-1e-stats.def ..
 
-
 clean:
-	rm -f *.aux *.bbl *.blg *.log *.dvi *.bak *.lof *.log *.lol *.lot *.out *.toc *.cut
+	$(RM) *.aux *.bbl *.blg *.log *.dvi *.bak *.lof *.log *.lol *.lot *.out *.toc *.cut
```
Comment (verdammelt) [inline rpg-1e-stats/Makefile:26]: These changes to use $(RM) are just stylistic

**Review by verdammelt — COMMENTED** [review]
@tunesmith added some comments in the Makefile in case you were curious what some of my changes were about.

**Pull request merged**